plugins/in_logcat/entities.h

```diff
@@ -0,0 +1,100 @@
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2015-2022 The Fluent Bit Authors
+ *  Copyright (C) 2005-2017 The Android Open Source Project
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#pragma once
+
+#include "stdint.h"
+
+/**
+ * Android log priority values, in increasing order of priority.
+ */
+typedef enum android_LogPriority
```

Comment:
I'm not entirely sure what the stance is when it comes to enums, I was under the impression that we didn't use them but I noticed they are used in some of the code. 

Typedefs on the other hand I think we don't usually use, there are a few here and there but I don't think the idea is to use them.